docs/PacketFence_AeroHIVE_Quick_Install_guide.asciidoc

```diff
@@ -0,0 +1,301 @@
++++++++++++++++++++++++++++++++++
+This step will configure the general options of your PacketFence installation. Theses are needed configurations that will most of the time fits customer specifications.
+
+Almost all of the required information here are self-explanatory. The only one that could be confusing is the DHCP Servers section. In this one, enter a comma-delimited list of all the DHCP Server on the customer network so when PacketFence will see DHCP trafic originating from these IPs, no rogue-dhcp alerts will be triggered.
+
+Click Continue once all the field are completed.
+
+Step 5: Administration
+++++++++++++++++++++++
+This is the step where you create the administrative user to access the PacketFence Administration Web Interface.
+
+Simply provide the desired username and password, then click Create user.
+
+Step 6: Services - Confirmation
++++++++++++++++++++++++++++++++
+The last but not the least. Here, you start the PacketFence server according to the configurations made in the previous steps. If everything goes as expected, you'll be prompted by a window inviting you to continue to the web administration interface.
+
+You'll be able to login to the PacketFence web administration interface with the credentials created in Step 4.
+
+Services status will help you monitor if everything goes as expected. If not, you'll see which service is in trouble and the log output will help you determine the problem that occurs.
+
+Configuring AeroHIVE controller in PacketFence
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Now that you have a fully functionnal PacketFence installation, you'll need to add the AeroHIVE AP to the PacketFence switches database for correct intergration.
+
+To do so, login to the PacketFence web administration interface if it is not already done. Click on the Configuration tab and select the switches section.
+
+We'll use the clone function to add a new switch (the AeroHive AP) to the PacketFence database. With the mouse pointer, go over the default switch and you'll see a set of icons appearing at the left of it. Click on the second one (a paper sheet with a plus sign).
+
+Confirgure the switch like the following:
+
+image::docs/images/aerohive-pf-config-1.png[scaledwidth="100%",alt="General switch settings"]
+
+image::docs/images/aerohive-pf-config-role.png[scaledwidth="100%",alt="Roles switch settings"]
+
+image::docs/images/aerohive-pf-config-snmp.png[scaledwidth="100%",alt="SNMP switch settings"]
+
+Modify your RADIUS secret.
+
+Click Add switch
+
+The newly added controller should shows up in the list.
+
+PacketFence configuration is done. You may now reboot the demonstation PC (leave the USB key in). Once rebooted, the web browser should open in the PacketFence web administation interface.
+
+Step 3: Configure AeroHIVE
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+This section will discuss about the configuration of your AeroHIVE AP in order to use it with our configured PacketFence environment.
```

Comment:
This section will help you configure your AeroHIVE AP for integration with PacketFence using the VLAN-based enforcement.